app/views/small_grants_fund/index.html.erb

```diff
@@ -160,15 +160,15 @@
       <div class="m-card -fullWidth">
         <h2 class="card-title">Resources and support</h2>
 
-        <a href="" class="card-title-subitle _bold _uppercase">small grand funds webinars</a>
+        <a href="/getinvolved/apply-to-the-small-grants-fund" class="card-title-subtitle _bold _uppercase">small grand funds webinars</a>
         <ul class="link-list">
-          <li><a href="">How to apply</a></li>
-          <li><a href="">Stories from the Field: 2015 projects</a></li>
+          <li><a href="/getinvolved/apply-to-the-small-grants-fund">How to apply</a></li>
```

Comment:
You are already in the new page of small grants fund (the other one is going to be removed), this is not correct. Maybe you should set the Guidelines for Aplicants link. You have already usedit